src/main.py

```diff
@@ -40,7 +43,18 @@ def main() -> None:
     """
 
     while True:
-        get_input()
+        #get imput 
+        for event in pg.event.get():
+            if event.type == pg.KEYDOWN:
+                if event.key == pg.K_UP or pg.K_SPACE:
+                    if Sneak == False
```

Comment:
Comparing a boolian to a boolian is redundant. rather write:
```suggestion
                    if not Sneak:
```